TESTING.md

```diff
@@ -1,121 +1,265 @@
-# Using `github.com/gravitational/teleport-plugins/lib/testing` package for writing tests
+# Testing access plugins
 
-This package implements Teleport integration testing.
+## What
 
-For the complete examples, please refer to [email plugin tests](access/email/email_test.go).
+The library which allows to run an access plugin integration tests using the standalone Teleport binary instead of the standard integration testing package.
 
-```go
-import . "github.com/gravitational/teleport-plugins/lib/testing"
-import "github.com/gravitational/teleport-plugins/lib/testing/integration"
-```
+## Why
 
-# Initialization
+Having Teleport as the dependency is hard for maintain. Teleport requires pre-built binary files to compile. Those files are generated in the Teleport `Makefile` before build. `go mod tidy` fails because it does not know anything about those preliminary steps.
```

Comment:
:nail_care: 
```suggestion
Having Teleport as the dependency is hard to maintain. Teleport requires pre-built binary files to compile. Those files are generated in the Teleport `Makefile` before build. `go mod tidy` fails because it does not know anything about those preliminary steps.
```